pyeudiw/trust/__init__.py

```diff
@@ -127,9 +140,12 @@ def _handle_x509_pem(self):
                 "a recognizable Trust Anchor."
             )
 
-        pem = trust_anchor['x509']['pem']
-
-        _is_valid = verify_x509_anchor(pem)
+        try:
+            pem = trust_anchor['x509']['pem']
+            _is_valid = verify_x509_anchor(pem)
+        except KeyError:
+            raise MissingTrustType(
+                f"Trust Anchor: '{trust_anchor_eid}' has no x509 trusst entity")
```

Comment:
ss -> s

Reply:
Resolved in [5a74ea0](https://github.com/italia/eudi-wallet-it-python/pull/188/commits/5a74ea0a4091d74e13d3975b140767d8b65ab94f)